Bindings/bindings.json

```diff
@@ -113,6 +113,26 @@
                     "label": "$eventHubTrigger_connection_label",
                     "help": "$eventHubTrigger_connection_help",
                     "placeholder": "[variables('selectConnection')]"
+                },
+                {
+                    "name": "cardinality",
+                    "value": "enum",
+                    "resource": "EventHub",
```

Comment:
Remove  "resource": "EventHub". This opening blades for resource selection like storage, eventhub connection